File: Lokalano-partnerstvo/client/src/app/shared/models/obuka.ts
export interface Obuka {
  id: number;
  naziv: string;
  aktivan: boolean;
  brojPolaznika: number;
  cijena: number;
  datumObjave: Date;
  datumPocetka: Date;
  obukaKategorija: string;
  obukaKategorijaId: number;
  objavio: string;
  opis: string;
  kratakOpis: string;
  trajanje: number;
  imageUrl: string;
  photo: Photo;
}
export interface Photo {
  id: number;
  imageUrl: string;
  fileName: string;
}
export interface ObukaToCreate {
  naziv: string;
  aktivna: boolean;
  brojPolaznika: number;
  cijena: number;
  datumObjave: string;
  datumPocetka: string;
  fokus: boolean;
  obukaKategorijaId: number;
  opis: string;
  kratakOpis: string;
  trajanje: number;
  imageUrl: string;
}

export class ObukaFormValues implements ObukaToCreate {
  naziv = '';
  aktivna = true;
  brojPolaznika = 0;
  cijena = 0;
  datumObjave = Date.now.toString();
  datumPocetka = Date.now.toString();
  fokus = false;
  obukaKategorijaId: number;
  opis = '';
  kratakOpis = '';
  trajanje = 0;
  imageUrl = '';

  constructor(init?: ObukaFormValues) {
    Object.assign(this, init);
  }
}

File: Lokalano-partnerstvo/Core/Entities/Obuka.cs
using System;

namespace Core.Entities
{
    public class Obuka : BaseEntity
    {
        public string Naziv { get; set; }
        public bool Aktivan { get; set; }
        public int BrojPolaznika { get; set; }
        public double Cijena { get; set; }
        public DateTime DatumObjave { get; set; }
        public DateTime DatumPocetka { get; set; }
        public int ObukaKategorijaId { get; set; }
        public ObukaKategorija ObukaKategorija { get; set; }
        public string Objavio { get; set; }
        public string Opis { get; set; }
        public string KratakOpis { get; set; }
        public int Trajanje { get; set; }
        public Photo Photo { get; set; }
        public int? PhotoId { get; set; }
        public string  ImageUrl { get; set; }

        public void AddPhoto(string imageUrl, string fileName)
        {
            var photo = new Photo
            {
                FileName = fileName,
                PictureUrl = imageUrl
            };

            Photo = photo;
        }
        public void RemovePhoto()
        {
            if (Photo != null)
            {
                Photo = null;
            }
        }
    }
}